orders/migrations/0003_static_orders.py

```diff
@@ -0,0 +1,79 @@
+from django.db import migrations, transaction
+from django.utils import timezone
+
+DOG_OWNER_ID = ['4', '5', '6', '7', '8', '9', '10', '11', '12', '13', '14', '15', '16', '17', '18', '19', '20', '21',
+                '22', '23', '4', '5', '6', '7', '8', '9', '10', '11', '12', '13', '14', '15', '16', '17', '18', '19',
+                '20', '21', '22', '23']
+
+DAYCARE_ID = ['4', '5', '6', '7', '8', '9', '10', '11', '12', '13', '14', '15', '16', '17', '18', '19', '20', '21',
```

Comment:
What's going on here, what are those id`s?

Reply:
Those are the id's of the static daycares and the static dog owners, from previous pull requests.
I used them to populate Hotails with orders from existed users.

Reply:
maybe for daycare you can try this   static_daycares = [daycare for daycare in DayCare.objects.all() if 'static' in daycare.user.username]
you then concat it with static_daycares[::-1]
** not so sure about the static dogonwers

Reply:
I'm not sure if depending on their name is better than depending on their id, because they're static, and they should remain as they are ATM.

Reply:
Maybe so, yet this is not so readable because we won't remember the ids, but if we call all the "static daycares" it will be clear for everyone

Reply:
After checking in the database about the static daycares, I see that they don't have the word "static" in their name, or anything else. I'd change it to the way you suggested for better readability, but apparently it can't be done.

![image](https://user-images.githubusercontent.com/88926592/166121590-cd2812c7-ae60-4a55-bcbd-5a93056f146e.png)

Reply:
Ok You're right. I'll change it.

Reply:
After trying to change it to the way you suggested, a problem occurred - because the migration of the daycares is in the same process of the migration of the orders, it can't be done.
Plus, the dog owners don't have any identifier like the daycares, so to be consistent, I'll keep the array of IDs for both daycares and dog owners.

Reply:
I agree with @Yuval-Vino,
about the dog-owner id I don't see that there is common information for any field like the 'static' prefix in daycare, so maybe run in a loop instead in the range of 4 to 23, maybe something like that:
`DOG_OWNER_ID = [id for id in range(4,24)]`

Reply:
I've changed the implementation of the lists according to your suggestions.
Thanks for the help guys! appreciate it :)